packages/flutter_goldens/lib/flutter_goldens.dart

```diff
@@ -43,7 +76,8 @@ abstract class FlutterGoldenFileComparator extends GoldenFileComparator {
   /// system and platform can be replaced by mock instances.
   @visibleForTesting
   FlutterGoldenFileComparator(
-    this.basedir, {
+    this.basedir,
+    this.skiaClient, {
```

Comment:
Is this ok to be null? If not assert and add it to the doc above.

May be good to mention this comment in the doc either way.